CMSgov/onemac
# Pr test

Story: Brandons Made up Agenda 02
Endpoint: https://d33javhgxcfrha.cloudfront.net/

### Details

Add an autogenerated comment with the endpoint url upon creation of a PR

### Changes

- created new github action workflow that acts only on pull_request

### Test Plan

1. Create a new PR on a branch that has been deployed
2. Verify a comment is generated with the endpoint url (workflow takes about 20 secs to complete)
